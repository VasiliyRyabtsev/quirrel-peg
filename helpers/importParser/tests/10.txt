#strict-compile:yes

// import b
import math // as math2
 import "dagor2.math" as m;
from "generic/std.nut" import types.TMatrix as Matrix, /* types.unknown as known */
  types.Point2 as point
/* comment
comment 
// */
from module2 import *


local a = require("x.nut")
return a.x
