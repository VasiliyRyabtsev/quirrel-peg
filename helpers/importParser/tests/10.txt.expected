math(math):
dagor2.math(m):
generic/std.nut(std): id=Matrix, path=types.TMatrix; id=point, path=types.Point2;
module2(module2): id=, path=*;
#strict-compile:yes